lib/src/types/item.ts

```diff
@@ -0,0 +1,41 @@
+import { Connector, ConnectorCredential } from './connector'
+
+/**
+ * Item status:
+ * UPDATED: The last sync process has completed succesfully and all new data is available to collect.
+ * OUTDATED: The parameters were correctly validated but there was an error in the last execution. It can be retried.
+ * LOGIN_ERROR: The connection must be updated to execute again, it won't trigger updates until the parameters are updated.
+ * WAITING_USER_INPUT: The connection requires user's input to continue the sync process, this is common for MFA authentication connectors
+ * UPDATING: An update process is in progress and will be updated soon.
+ */
+export enum ItemStatus {
+  LOGIN_ERROR = 'LOGIN_ERROR',
+  OUTDATED = 'OUTDATED',
+  UPDATED = 'UPDATED',
+  UPDATING = 'UPDATING',
+  WAITING_USER_INPUT = 'WAITING_USER_INPUT',
+}
+
+export type Item = {
+  /** primary identifier of the Item */
```

Comment:
FYI, the one-liner comment format that I've been using that is kept after TSC build, is this: 
```
export type PluggyConnectProps = {
  /*! client authorized API key */
  connectToken: string;
  /*! whether to include sandbox connectors or not */
  includeSandbox?: boolean;
}
```

I'm not sure about the advantages/disadvantages of using a one-line comment block `/** my comment */` compared to the above though. 
Only I can note that visually the IDE colors the `/** asd */` as a multiline code-block (green color - same as a "TODO" remark in a comment), VS the other `/*! asd */` as gray like a regular single line comment.